packages/argo-checkout/src/extension-points/api/post-purchase/post-purchase.ts

```diff
@@ -109,3 +102,104 @@ interface Metafield {
   value: string | number;
   valueType: 'integer' | 'string' | 'json_string';
 }
+
+interface ExplicitDiscount {
+  value: number;
+  valueType: ExplicitDiscountType;
+  title: string;
+}
+
+interface AddVariantChange {
+  type: ChangeType.AddVariant;
+  variantId: number;
+  quantity: number;
+  discount?: ExplicitDiscount;
+}
+
+type Change = AddVariantChange;
```

Comment:
this is eventually gonna be something like `type Change = AddVariantChange | SomeOtherChange | YetAnotherChange;